pkgs/development/python-modules/pywick/default.nix

```diff
@@ -1,16 +1,27 @@
 { buildPythonPackage
 , fetchFromGitHub
 , pythonOlder
-, pytest
+, pytestCheckHook
+, albumentations
```

Comment:
there's no albumentations in nixpkgs

```
error: Function called without required argument "albumentations"
```

Reply:
How did this evaluate on ofborg!?

Reply:
@SuperSandro2000 in fact, despite ofborg being green on x64-linux,

```The following builds were skipped because they don't evaluate on x86_64-linux: python310Packages.pywick```

I knew that albumentations and prodict weren't in Nixpkgs and I'm sure I had `pywick` marked broken as per the first comment on this PR -- no idea how that didn't make it into `master` ?!